charts/workflow/requirements.yaml

```diff
@@ -45,6 +45,7 @@ dependencies:
     version: <registry-token-refresher-tag>
     repository: https://charts.deis.com/registry-token-refresher
   - name: router
+    condition: !global.experimental_native_ingress
```

Comment:
So the plan is to include the router even if the user opts in to native ingress?